bindings/python/py_src/tokenizers/implementations/sentencepiece_bpe.py

```diff
@@ -32,8 +32,9 @@ def __init__(
             tokenizer.add_special_tokens([str(unk_token)])
 
         tokenizer.normalizer = NFKC()
-        tokenizer.pre_tokenizer = pre_tokenizers.Metaspace(replacement=replacement, add_prefix_space=add_prefix_space)
-        tokenizer.decoder = decoders.Metaspace(replacement=replacement, add_prefix_space=add_prefix_space)
+        prepend_scheme = "always" if add_prefix_space else "never"
```

Comment:
oh perfect

Reply:
We could make it BC in python only if the python layer of MetaSpace just use `prepend_scheme = "always" if add_prefix_space else "never"`. WOuld be nice